Replace outdated SYCL_DEVICE_FILTER with ONEAPI_DEVICE_SELECTOR (#484)
oneapi-src/oneMKL: examples/README.md

# Using single precision (float) data type
#
# Device will be selected during runtime.

> # The environment variable ONEAPI_DEVICE_SELECTOR can be used to specify
> # ONEAPI device

Reviewer: ```suggestion
# The environment variable ONEAPI_DEVICE_SELECTOR can be used to specify
# available devices
```

# Using single precision (float) data type
#
# Device will be selected during runtime.
# The environment variable ONEAPI_DEVICE_SELECTOR can be used to specify

> # ONEAPI device

Reviewer: One last place:
```suggestion
# available devices
```

Author: Thanks for the catch, made the change. Resolving

# Using single precision (float) data type
#
# Device will be selected during runtime.

> # The environment variable ONEAPI_DEVICE_SELECTOR can be used to specify
> # SYCL device

Reviewer: ```suggestion
# The environment variable ONEAPI_DEVICE_SELECTOR can be used to specify
# available devices
```
